axis/resonant_axis.f95

```diff
@@ -0,0 +1,111 @@
+module resonant_axis
+
+use global_parameters
+implicit none
+
+contains
+
+!----------------------------------------------------------------------------------------------
+subroutine make_id_matrix_2body(pl_name,max_order)
+! Creates id. matrix (twobody case)
+! Given:
+!   pl_name - planet name IN CAPITALS
+!   max_order - maximum resonance order
+! Produces:
+!   file with idmatrix_2body
+    integer:: pl_id,un,max_order
+    integer:: m1,m
+    character(8):: pl_name
+    character(14):: co='(4i4,f23.16)'
+    integer,dimension(4):: resonance
+    
+    pl_id=planet_id(pl_name)
+    un=8+pl_id
+    open(unit=un,file="id_matrix_"//trim(pl_name)//".dat",status='replace')
+    ! Get id. matrix for a planet by a given maximum order
+    do m1=1,max_order
+        do m=-1,-m1-max_order,-1
```

Comment:
Why m is negative only? 
5J+2 — IMO, normal resonance, is not it? In your case we check only 5J-2, right?